docs/installation/architecture.rst

```diff
@@ -2,3 +2,10 @@
 
 KSQL Architecture
 -----------------
+
+KSQL is an abstraction of Kafka Streams API.
+
+KSQL consumes streams of data that are stored in Kafka topics. KSQL data can be structured as Avro, JSON, DELIMITED (CSV).
+
+There are KSQL servers, which form KSQL clusters, and there is a CLI client that connect to the KSQL clusters.
```

Comment:
I thought we were moving away from the 'KSQL Cluster' terminology. Below we use 'KSQL server pools'. Perhaps it would be better to converge on that term.

Reply:
@miguno ^^